src/main/java/nl/tudelft/dnainator/ui/widgets/AboutDialog.java

```diff
@@ -0,0 +1,82 @@
+package nl.tudelft.dnainator.ui.widgets;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import javafx.scene.Node;
+import javafx.scene.control.Alert;
+import javafx.scene.control.ButtonBar.ButtonData;
+import javafx.scene.control.ButtonType;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+
+/**
+ * Creates a {@link Alert} displaying information about the
+ * application.
+ */
+public class AboutDialog {
```

Comment:
Can't we just extend `Alert`? (Should also apply this to `ExceptionDialog` then)